riscv/csrs.cc

```diff
@@ -975,6 +975,8 @@ bool counter_proxy_csr_t::myenable(csr_t_p counteren) const noexcept {
 }
 
 void counter_proxy_csr_t::verify_permissions(insn_t insn, bool write) const {
+  proxy_csr_t::verify_permissions(insn, write);
```

Comment:
What is the effect of this? Was there a bug here?

If this is not related to Smstateen, it does not belong in this PR.

Reply:
I think it's a bug here. I'll send another PR later.